src/cli/config/config.go

```diff
@@ -209,14 +220,24 @@ func writeRepoConfigWithViper(vpr *viper.Viper, s3Config storage.S3Config, m365C
 	return nil
 }
 
+// GetStorageAndAccount creates a storage and account instance by mediating all the possible
+// data sources (config file, env vars, flag overrides) and the config file.
+func GetStorageRepoAndAccount(ctx context.Context) (storage.Storage, account.Account, string, error) {
+	return getStorageAndAccountWithViper(GetViper(ctx), true, make(map[string]string))
+}
+
 // GetStorageAndAccount creates a storage and account instance by mediating all the possible
 // data sources (config file, env vars, flag overrides) and the config file.
 func GetStorageAndAccount(
 	ctx context.Context,
 	readFromFile bool,
 	overrides map[string]string,
-) (storage.Storage, account.Account, error) {
-	return getStorageAndAccountWithViper(GetViper(ctx), readFromFile, overrides)
+) (storage.Storage,
```

Comment:
Style standards: similar to the func declaration, multi-line return values should give each param a new line.
```suggestion
) (
    storage.Storage,
```